geonode/layers/layer_uploader.py

```diff
@@ -0,0 +1,59 @@
+from django.contrib.auth.decorators import login_required
+
+__author__ = 'Irwan Fathurrahman <irwan@kartoza.com>'
```

Comment:
Change file header to standard geonode header, like in other files, and follow the convention.